tests/tests_unit/dummyconfig_keyvault.yaml

```diff
@@ -0,0 +1,25 @@
+version: "1"
+
+logger:
+  console:
+    level: INFO
+
+azure-keyvault:
+    authentication_method: client-secret
+    client_id: ${KEYVAULT_CLIENT_ID}
+    tenant_id: ${KEYVAULT_TENANT_ID}
+    client_secret: ${KEYVAULT_CLIENT_SECRET}
+    keyvault_name: extractor-credentials
```

Comment:
Some nitpick:

* Make these use hyphen-casing instead of snake_casing
* Rename `tenant_id` to `tenant` and `client_secret` to `secret` so that we use the same keys as in `idp-authentication`